.github/workflows/release.yml

```
@@ -21,3 +21,96 @@
+  
+            asyncio.get_event_loop().run_until_complete(main())
+  
+            " > script.py
+
+        - name: Run Python script
+          shell: bash -l {0}
+          run: |
+            conda activate test-env
+            python3 script.py
+
+    test-pip:
```

actionlint

unexpected key "test-pip" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)